week2/homework/src/index.js

```diff
@@ -1,3 +1,98 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+let fs = require('fs');
+let command = process.argv[2];
+let index = process.argv[3];
```

Comment:
`index` is not the best variable name, since this is not always the index, in the case of `add` for example it is the actual text of the todo item...